proposals/btco-vc/btco_vc_proposal.md

```diff
@@ -0,0 +1,96 @@
+# Bitcoin Ordinals Verifiable Credentials Framework
+
+## Metadata
+- Proposal Author(s): [Brian Richter](https://github.com/brianorwhatever)
+- Chair(s): Brian Richter
+- Status: [Proposed]()
+- Since: November 28, 2024
+- Tags: bitcoin, ordinals, verifiable-credentials, did
+- Expected End Date: January 31, 2025
+- Version: 0.0.1
+
+[//]: # (Do not fill out below. To be filled out by chairs post-approval)
+- Start Date: <date>
+- URI: <uri>
+
+## Summary
+
+This proposal outlines a framework for implementing verifiable credentials on Bitcoin using Ordinal inscriptions. The framework builds on the `did:btco` method, which associates DIDs with specific ordinals and implements DID Linked Resources, enabling tamper-proof and decentralized verification of digital artifacts without intermediaries. The inherent content-addressing nature of Ordinal inscriptions makes them particularly well-suited for implementing DID Linked Resources. The project aims to standardize these capabilities for various use cases including event tickets, digital collectibles, ownership records, and Verifiable AI Inference (VAI) credentials that prove the authenticity and origin of AI-generated content.
+
+## Motivation
+
+Bitcoin's Ordinal inscriptions provide a unique opportunity to create immutable, publicly verifiable digital artifacts. However, there is currently no standardized way to use this capability for verifiable credentials. The permanent and content-addressed nature of inscriptions creates an ideal foundation for implementing DID Linked Resources, as each inscription naturally serves as a globally addressable resource. This framework aims to:
+
+- Enable creation of tamper-proof verifiable credentials directly on Bitcoin
+- Provide a standardized approach for digital artifact verification
+- Leverage Bitcoin's security and decentralization for credential issuance
+- Create interoperable standards for credential verification
+- Address privacy considerations in public blockchain deployments
+- Utilize inscriptions' content-addressing capabilities for robust DID Linked Resources
```

Comment:
I think we have a few experts with the expert/mentors that might be able to help. We'll get back to you on some thoughts on who can help and how?